calculator.py

```diff
@@ -40,45 +40,118 @@
 
 
 """
+# import pysnooper
 
-
+# @pysnooper.snoop()
 def add(*args):
     """ Returns a STRING with the sum of the arguments """
 
-    # TODO: Fill sum with the correct value, based on the
+    # TODONE: Fill sum with the correct value, based on the
     # args provided.
-    sum = "0"
 
-    return sum
+    my_sum = sum(map(int,args))
+
+    return str(my_sum)
+
+# TODONE: Add functions for handling more arithmetic operations.
+
+def subtract(*args):
+    """ Returns a STRING with the difference of the arguments """
+
+
+    a = int(args[0])
```

Comment:
You also want to be able to handle multiple arguments here, don't assume that you will only ever have 2 arguments. Ex: `/subtract/10/5/1`